src/app/pages/audit/components/audit/audit.component.ts

```diff
@@ -237,6 +266,8 @@ export class AuditComponent implements OnInit, AfterViewInit, OnDestroy {
       this.dataProvider.setParams([(query as AdvancedSearchQuery<AuditEntry>).filters]);
     }
 
+    this.advancedSearchAutocomplete.showDatePicker$.next(false);
```

Comment:
This breaks encapsulation of advanced search component. Audit component shouldn't care about internal details of advanced search autocomplete. There should be some other way to implement this, perhaps by closing date picker when focus is lost or something like this.